config/stage_test/template_params.yaml

```diff
@@ -5,3 +5,6 @@
   name: IMAGE_TAG
   value: ''
   required: true
+- description: Pull secrets for deploying image
+  name: IMAGE_PULL_SECRET_RH_REGISTRY
```

Comment:
Just a nitpick: The name is very specific to concrete registry. Alternatives that I can see can be: `IMAGE_PULL_SECRET_NAME`, `SA_IMAGE_PULL_SECRET`, etc...

Reply:
Changes made, please let me know if it is OK now,

Reply:
All good! thx